packages/jest-expo/config/getPlatformPreset.js

```diff
@@ -31,7 +41,11 @@ function getPlatformPreset(displayOptions, extensions, platform, isServer) {
 
   const upstreamBabelJest = getUpstreamBabelJest(expoPreset.transform) ?? '\\.[jt]sx?$';
 
-  return withWatchPlugins({
+  if (isReactServer && displayOptions && displayOptions.name) {
+    displayOptions.name = `rsc/${extensions[0]}`;
```

Comment:
Should this completely clobber displayOptions.name instead of appending to it?

Reply:
Yes, the name should be a fixed value.